packages/video_player_avplay/lib/video_player.dart

```diff
@@ -308,6 +312,11 @@ class VideoPlayerController extends ValueNotifier<VideoPlayerValue> {
   /// Only for [VideoPlayerController.network].
   final Map<String, dynamic>? playerOptions;
 
+  /// Sets specific feature values for HTTP, MMS, or specific streaming engine (Smooth Streaming, HLS, DASH, DivX Plus Streaming, or Widevine).
+  /// The available streaming properties depend on the streaming protocol or engine.
+  /// Only for [VideoPlayerController.network].
+  final Map<String, dynamic>? streamingProperty;
```

Comment:
Are there any plan for additional property types besides `ADAPTIVE_INFO`?
According to https://developer.samsung.com/smarttv/develop/api-references/samsung-product-api-references/avplay-api.html, String type seems to be sufficient.

What about `Map<String, String>`?

Reply:
Ok